be/test/column/binary_column_test.cpp

```diff
@@ -655,25 +655,14 @@ PARALLEL_TEST(BinaryColumnTest, test_replicate) {
     ASSERT_EQ("def", slices[4]);
 }
 
-PARALLEL_TEST(BinaryColumnTest, test_element_memory_usage) {
+PARALLEL_TEST(BinaryColumnTest, test_reference_memory_usage) {
     auto column = BinaryColumn::create();
     column->append("");
     column->append("1");
     column->append("23");
     column->append("456");
 
-    ASSERT_EQ(22, column->Column::element_memory_usage());
-
```

Comment:
change to `bytes_usage`?

Reply:
> change to `bytes_usage`?

test_byte_size method is already existed, I think we don't need to check it in this method.